go.mod

```diff
@@ -1,10 +1,21 @@
 module github.com/certusone/solana_exporter
 
-go 1.13
+go 1.22
```

Comment:
Just pull this in from your rebase.

Reply:
The Go Update PR was to merge that branch into this branch